File: cs50/pset1/mario/mario.c
// Problem Set 1: Mario Pyramid

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt for height
    int height;
    do
    {
        height = get_int("Height (1 thru 8): ");
    }
    while (height < 1 || height > 8);

    // Print pyramid
    for (int i = 1; i <= height; i++)
    {
        for (int k = height - i; k > 0; k--)
        {
            printf(" ");
        }

        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: cs50/pset6/mario/mario.py
#!/usr/bin/env python3
# Print half pyramid from Mario
from cs50 import get_int

# Get pyramid height from user; must be between 1 and 8 inclusive
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

for i in range(1, height + 1):
    for j in range(height - i):
        print(" ", end="")
    for k in range(i):
        print("#", end="")
    print()